src/backend/backend.h

```diff
@@ -37,20 +37,102 @@ namespace snmalloc
   template<
     SNMALLOC_CONCEPT(ConceptPAL) PAL,
     bool fixed_range,
-    typename PageMapEntry = MetaEntry>
+    template<typename> class PageMapEntry = MetaEntry>
```

Comment:
Maybe this is the right time to remove this templating. As discussed, subclassing is not the right path here and it is the only use-case for this template.